client/sites-dashboard/components/sites-site-launch-nag.tsx

```diff
@@ -106,7 +92,8 @@ export const SiteLaunchNag = ( { site }: SiteLaunchNagProps ) => {
 				recordTracksEvent( 'calypso_sites_dashboard_site_launch_nag_click' );
 			} }
 		>
-			<SiteLaunchDonut /> <SiteLaunchNagText>{ text }</SiteLaunchNagText>
+			<SiteLaunchDonut numberOfSteps={ numberOfSteps } completedSteps={ completedSteps } />
```

Comment:
It's a little more complicated than just using `numberOfSteps` and `completedSteps`. The focused launchpad and card version of the launchpad use different math.

The focused launchpad actually subtracts launch tasks from the counts.

https://github.com/Automattic/wp-calypso/blob/3447e403c5d8f8749c882b0fe8c338720e05f4b8/client/my-sites/customer-home/components/full-screen-launchpad.tsx#L130-L131

Which could lead to subtly different donuts. E.g. my focused launchpad is at 5/5 but my dashboard donut is at 5/6

![CleanShot 2025-03-10 at 12 04 17@2x](https://github.com/user-attachments/assets/ab92f241-cf56-4db4-bd88-89a56210e86f)

![CleanShot 2025-03-10 at 12 05 13@2x](https://github.com/user-attachments/assets/32557f0b-0e10-48c7-bb09-fd0f668fb0a6)

Reply:
Actually I've asked about this case https://github.com/Automattic/wp-calypso/issues/100260#issuecomment-2709129315